src/OrchardCore.Modules/Orchard.Autoroute/Handlers/AutoroutePartHandler.cs

```diff
@@ -100,7 +100,11 @@ public override void Updated(UpdateContentContext context, AutoroutePart part)
 
             if (!String.IsNullOrEmpty(pattern))
             {
-                part.Path = _tokenizer.Tokenize(pattern, new Dictionary<string, object> { ["Content"] = part.ContentItem });
+                var templateContext = new TemplateContext();
+                templateContext.SetValue("ContentItem", part.ContentItem);
```

Comment:
Why? to have a constructor instead of a method call? Later we'll decide to provide the ContentPart too in the context, we'll have to create another class.
I don't really see any value if just not using the `"ContentItem"` string directly.